python/tests/compat/pandas_compat_test.py

```diff
@@ -0,0 +1,63 @@
+import pandas
+import scipp
+from scipp.compat.pandas_compat import from_pandas
+
+
+def test_1d_dataframe():
+    pandas_df = pandas.Series(data=[1, 2, 3])
+
+    sc_dataarray = from_pandas(pandas_df)
+
+    assert (sc_dataarray.values == scipp.array(dims=["pandas_row"],
+                                               values=[1, 2, 3]).values).all()
```

Comment:
Use
```suggestion
    assert sc.identical(sc_dataarray, scipp.array(dims=["pandas_row"], values=[1, 2, 3]))
```
This will also ensure that unit, dimension labels, coords, ... are correct. Same goes for everything below, you should basically never `assert` using the `values` or `value` property.